client/src/FileManager.module.css

```diff
@@ -1,37 +1,57 @@
 .FileManagerWrapper {
     width: 100%;
-    height: calc( 100vh - 108px - 60px - 24px - 56px - 2px - 10px );
+    height: calc(100vh - 108px - 60px - 24px - 56px - 2px - 10px);
 }
+
 .FileManager {
     width: 100%;
     height: 100%;
     border: none;
 }
 
-.ButtonGroup :first-child {
-    border-left-color: transparent;
+.FileToolBar {
+    height: 87px;
+    background-color: #e5e5e5;
+}
+
+.ButtonGroupWrapper {
+    padding-left: 24px;
+    padding-right: 24px;
+    height: 65px;
+    gap: 12px;
 }
-.ButtonGroup :last-child {
-    border-right-color: transparent;
+
+.ButtonGroupWrapper:not(:first-of-type) {
+    border-left: 1px solid #BEBEBE;
 }
-.ButtonGroup > *, .ButtonGroup > *:disabled {
-    border-left: 1px solid var(--gray);
-    border-right: 1px solid var(--gray);
+
+.ButtonGroupTitle {
+    margin: 0;
+    color: #3d3d3d;
+    line-height: 16px;
+    font-size: 16px;
```

Comment:
As shown in the screenshot above, would be better to define font size in rem so users can increase font size if they need to